### ISSUE TITLE
Intro to Neural Networks - 12. Implementing gradient descent - code pass the test wo sigmoid function

### ISSUE THREAD

**Issue by Adosha** [udacity/dlnd-issue-reports#33]

**Comment by Adosha** [comment]
Code passes the test if you will use  "output = np.dot(x, weights)"  -- without  sigmoid function.

**Comment by Adosha** [comment]
also typo in solution.py:

        # The error, the target minues (!!!!!!) the network output

**Comment by oritavital** [comment]
typo was fixed

**Comment by Brok-Bucholtz** [comment]
We've added this to our backlog.
